pom.xml

```diff
@@ -66,7 +72,6 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>3.5.2</version>
```

Comment:
Why is the version removed? Common plugins should have their versions defined here.